src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav {
+  &__links {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link-item:not(:last-child) {
+    margin-right: 20px;
+  }
+  &__link {
+    position: relative;
+    line-height: 60px;
```

Comment:
The 'line-height: 60px;' seems to be a magic number. It's better to use variables for such values to maintain consistency and readability across the stylesheet.